### ISSUE TITLE
PULL REQUESTS ARE WELCOME

### ISSUE THREAD

**Issue by Webkadabra** [Brainfock/Brainfock#8]
Hey guys! I'm really happy to see that "traffic" page on github stats shows that we get at least couple of clones each day, and since Brainfock is not updated for the latest Babel, I'm hoping that your're doing something to run Brainfock, so, please, share your code updates! I really hope to see our project grow to be something really useful for small teams at the very least.

— Sergii Gamainov

**Comment by thousif** [comment]
hey @Webkadabra, I would like to contribute to this project! I am having trouble setting up the project! I'm on a windows machine!, Is there anything that i should be aware of?

**Comment by jacksolutions** [comment]
is this project dead? i see no updates since very long time?

**Comment by Webkadabra** [comment]
@jacksolutions yes, it is. It was a very good code up until node 0.4. So if you have old node you probably can still run it

**Comment by Webkadabra** [comment]
I did not have money to fucking renew my domain even, fuck open source big time

**Comment by jacksolutions** [comment]
oh thats very sad to hear :( how can i help @Webkadabra 

or do you mind if i lead this project further? i will try to run it  or port it on new node version.

**Comment by jacksolutions** [comment]
give me your email please and lets discuss further on it?

**Comment by Webkadabra** [comment]
@jacksolutions be my guest, brother, if you could update it so it works.... I might even join you some time. My email is devkadabra@gmail.com

**Comment by Webkadabra** [comment]
@jacksolutions I could not even get any $ donations because I think PayPal is no allowing me to accept money, because I live in Ukraine, why would I need money, hell, everything is free here

**Comment by jacksolutions** [comment]
@Webkadabra you have good sense of humor sergii, i liked it, i think it will be a fun working together :)

**Comment by jacksolutions** [comment]
i understand the frustration sergii but i think paypal is now in ukraine as well?

**Comment by jacksolutions** [comment]
i dropped you an email, please let me know thanks

**Comment by Webkadabra** [comment]
@jacksolutions No, PayPal is run by gay faggots - I still can't get donations in Ukraine.
